src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -71,6 +71,14 @@ public static int Abs(int value)
             return value;
         }
 
+#if CORECLR
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static float Abs(float x)
```

Comment:
This should not be public. I am surprised that the build is not failing on this.